Binary files wxorig/.DS_Store and wxnew/.DS_Store differ
diff -uNr wxorig/src/generic/listctrl.cpp wxnew/src/generic/listctrl.cpp
--- wxorig/src/generic/listctrl.cpp	2009-11-19 19:58:02.000000000 -0500
+++ wxnew/src/generic/listctrl.cpp	2009-11-19 19:55:12.000000000 -0500
@@ -4310,6 +4310,7 @@
 #endif
 
         GetSizer()->Prepend( m_headerWin, 0, wxGROW );
+        GetSizer()->SetItemMinSize( m_headerWin, wxSize(-1,m_headerHeight) );
     }
     else
     {
